### PR TITLE
web-ui: Display test-pack git clone url in automated view

### LINKED ISSUE #38: web-ui: Show the git clone URL

### PR DESCRIPTION
**Pull request by drothlis**
Fixes #38.

**TODO:**

- [x] e2e-tests

### CODE REVIEW

**Comment by drothlis** [comment]
It looks like this:

screenshot

**Comment by wmanley** [comment]
This will be very useful.  Could you make it look more commandy?  Maybe monospace font?  Or possibly copy what github's style:
image

**Comment by drothlis** [comment]
How about this:

screenshot-1

When you click or tab into the input, it automatically selects the whole url:

screenshot-1-selected

**Comment by drothlis** [comment]
Or alternately:

screenshot-2

**Comment by drothlis** [comment]
Maybe the text should say "test-pack git URL" or something like that.

**Comment by wmanley** [comment]
I like single line, with the current text.

File: web-ui/app/views/automated.html [hunk 1]
```diff
@@ -1,5 +1,10 @@
 <form name="form" novalidate>
   <div class="form-group">
+    <div id="git-clone-url" class="help-block pull-right">
+      git clone URL:<br>
+      <input type="text" value="{{ test_pack.clone_url }}" select-on-focus />
```
Comment (wmanley) [inline web-ui/app/views/automated.html]: Should this be read-only?

**Comment by drothlis** [comment]
Addressed all your comments: It's on a single line, and the text input is read-only. This is ready to merge once CI gives the go-ahead.

**Pull request merged**